app/controllers/admin/r_and_r_items_controller.rb

```diff
@@ -0,0 +1,140 @@
+class Admin::RAndRItemsController < AdminController
```

Comment:
Add a comment saying this is based off the other controller it's based off of

Reply:
Done.